modules/virt-example-vlan-nncp.adoc

```diff
@@ -39,7 +39,7 @@ spec:
 <1> Name of the policy.
 <2> Optional: If you do not include the `nodeSelector` parameter, the policy applies to all nodes in the cluster.
 <3> This example uses a `hostname` node selector.
-<4> Name of the interface.
+<4> Name of the interface. For VLANs, only the `<interface_name>.<vlan_number>` format is supported.
```

Comment:
This is only true for baremetal I think, since the limitation is related to "ironic", but here it looks like a global limitation.

Reply:
Done. @qinqon Can you have another look?